client/me/purchases/manage-purchase/plan-details/index.jsx

```diff
@@ -41,22 +43,27 @@ class PurchasePlanDetails extends Component {
 			return null;
 		}
 
-		if ( pluginList.length < 1 ) {
-			return null;
-		}
+		const headerText = translate( '%(plan)s Plan', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( '%(planName)s Plan' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).